x/nexus/keeper/transfer.go

```diff
@@ -84,27 +135,42 @@ func (k Keeper) EnqueueForTransfer(ctx sdk.Context, sender exported.CrossChainAd
 		return 0, fmt.Errorf("recipient's chain %s does not support foreign assets", recipient.Chain.Name)
 	}
 
+	// merging transfers below minimum for the specified recipient
+	insufficientAmountTransfer, found := k.getTransferForRecipientAndAsset(ctx, recipient, asset.Denom, exported.InsufficientAmount)
+	if found {
+		asset = asset.Add(insufficientAmountTransfer.Asset)
+		k.deleteTransfer(ctx, insufficientAmountTransfer)
+	}
+
 	// collect fee
-	if feeDue := sdk.NewDecFromInt(asset.Amount).Mul(feeRate).TruncateInt(); feeDue.IsPositive() {
-		k.addTransferFee(ctx, sdk.NewCoin(asset.Denom, feeDue))
-		asset = asset.SubAmount(feeDue)
+	fees, _ := k.ComputeTransferFee(ctx, sender.Chain, recipient.Chain, asset)
```

Comment:
why ignore the error here?

Reply:
It's not an error, just some additional fee info useful for a query.